pihole

```diff
@@ -62,6 +62,12 @@ function setupLCDFunction {
     exit 1
 }
 
+function queryFunc {
+    domain=$2
+    for list in /etc/pihole/list.*;do echo $list;grep ${domain} $list;done
+    exit 1
```

Comment:
All the function exit with a return code of 1, they aren't exposed to the user, just internal. Not sure where the precedent was set for using 1 for all the functions, but we're just sticking with our convention for now.